sbi/analysis/__init__.py

```diff
@@ -5,5 +5,10 @@
     eval_conditional_density,
 )
 from sbi.analysis.gradient_ascent import gradient_ascent
-from sbi.analysis.plot import conditional_pairplot, pairplot
+from sbi.analysis.plot import (
+    conditional_pairplot,
```

Comment:
these imports appear to be unsorted, you may want to run `isort` on all changed files.